src/Apps/__test__/Fixtures/Artist/Routes/ArticlesFixture.ts

```diff
@@ -0,0 +1,216 @@
+export const ArticlesFixture = {
+  artist: {
```

Comment:
The following sequence of fixtures can be trimmed a bit but i like that its real data.